src/components/transformation/create/index.tsx

```diff
@@ -168,54 +122,25 @@ function TransformationCreate({ postWindowOpen }: Props) {
-                                error={!!entityNameError}
-                                helperText={entityNameError}
-                                value={derivationName}
-                                onChange={handlers.evaluateCatalogName}
-                                InputProps={{
-                                    startAdornment: (
-                                        <InputAdornment position="start">
-                                            {prefixOptions.length === 1 ? (
-                                                prefixOptions[0]
-                                            ) : (
-                                                <>
-                                                    <Select
-                                                        size="small"
-                                                        variant="standard"
-                                                        value={catalogPrefix}
-                                                        onChange={(evt) => {
-                                                            setCatalogPrefix(
-                                                                evt.target.value
-                                                            );
-                                                        }}
-                                                    >
-                                                        {prefixOptions.map(
-                                                            (prefix) => (
-                                                                <MenuItem
-                                                                    key={prefix}
-                                                                    value={
-                                                                        prefix
-                                                                    }
-                                                                >
-                                                                    {prefix}
-                                                                </MenuItem>
-                                                            )
-                                                        )}
-                                                    </Select>
-
-                                                    <Divider orientation="vertical" />
-                                                </>
-                                            )}
-                                        </InputAdornment>
-                                    ),
+                            <Box
+                                sx={{
+                                    p: 2,
                                 }}
-                            />
+                            >
+                                <PrefixedName
+                                    standardVariant
+                                    size="medium"
```

Comment:
**Personal Note:** all instances of `PrefixedName` in this component should have a size of `small`. Future rollback.

Reply:
I don't think so - in the old design a small input looks pretty weird. That's why it is set to medium.